tests/api/test_auth_api.py

```diff
@@ -8,99 +8,104 @@
 from tests.api.test_api_base import ApiTestBase
 from tests.test_helper import TestHelper
 from yuansfer.api_helper import APIHelper
-from yuansfer.api.offline_api import OfflineApi
+from yuansfer.api.auth_api import AuthApi
```

Comment:
what's the reason of changing this?  Are we no longer use `OfflineApi`?

Reply:
it supposed to be the unit test file for the auth API not the offline